apps/protocol-frontend-e2e/src/support/commands.ts

```diff
@@ -41,4 +41,140 @@ Cypress.Commands.add('interceptGQL', (httpMethod, operationNames) => {
       }
     });
   }
+
+});
+
+// Seed DAOs/Guilds into DB
+Cypress.Commands.add('seedDB',(tableName)=>{
```

Comment:
This command is probably doing too much it would probably be better to structure this. Instead of using a command it is probably better to create functions for such as create_guild() or create_contribution that are called in the before each hook. And then have a separate delete_guild or delete_contribution function.

Reply:
I see.  Just to clarify, I create a `**.ts` file under path `protocol-frontend-e2e/src/support` where I can have these functions?